### PR TITLE
add deriv-go page

### PR DESCRIPTION
**Pull request by fiona-deriv**
# Description
creating a new page

Summary

Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ x ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/deriv-go/components/_dgo-hero.js [hunk 1]
```diff
@@ -0,0 +1,315 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import { localize } from 'components/localization'
+import { Flex } from 'components/containers'
+import { Header, QueryImage } from 'components/elements'
+import { LinkButton } from 'components/form'
+import device from 'themes/device.js'
+import Dgoogle_play_Logo from '/home/deriv/Desktop/deriv-com/src/images/common/derivgo/derivgo_gogleplay.png'
```
Comment (yashim-deriv) [inline src/pages/deriv-go/components/_dgo-hero.js:10]: This asset seems to be breaking the build. Please include the `derivgo_gogleplay.png` image file as well

File: src/pages/deriv-go/components/_dgo-hero.js [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+    //     }
+    //     @media ${device.mobileL} {
+    //         max-width: 250px;
+    //         min-height: 244px;
+    //     }
+    //     @media ${device.mobileM} {
+    //         max-width: 205px;
+    //         min-height: 0;
+    //     }
+    // `
```
Comment (yashim-deriv) [inline src/pages/deriv-go/components/_dgo-hero.js:251]: Any reason why the code above is commented?

File: src/pages/deriv-go/components/_dgo-hero.js [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+    const DLogo = styled.img`
+        width: 213px !important;
```
Comment (yashim-deriv) [inline src/pages/deriv-go/components/_dgo-hero.js:253]: Generally, !important should be avoided unless absolutely necessary

File: src/pages/deriv-go/components/_new-app.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+const query = graphql`
+    query {
+        dgo_logo: file(relativePath: { eq: "derivgo/dgo_logo.png" }) {
+            ...fadeIn
+        }
+        qr_code: file(relativePath: { eq: "derivgo/qr_code.png" }) {
+            ...fadeIn
+        }
+    }
+`
```
Comment (yashim-deriv) [inline src/pages/deriv-go/components/_new-app.js:51]: These image files need to be included as well

File: src/pages/deriv-go/index.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+import Loadable from '@loadable/component'
+import DHero from './components/_dgo-hero'
+import { SEO } from 'components/containers'
+import { localize, WithIntl, Localize } from 'components/localization'
+import Copyright from 'components/layout/copyright.js'
+import { size } from 'themes/device'
+import { isBrowser } from 'common/utility'
+import dgo_logo from 'images/common/derivgo/derivgo_logo.png'
+import DGoBGMobile from 'images/svg/dtrader-bg-mobile.svg'
+import DGoBG from 'images/common/derivgo/derivgo_background.png'
```
Comment (yashim-deriv) [inline src/pages/deriv-go/index.js:11]: The files above need to be included as well

**Review by yashim-deriv — CHANGES_REQUESTED** [review]

File: src/pages/deriv-go/components/_dgo-hero.js (hunk 2 above)
Comment (fiona-deriv) [reply src/pages/deriv-go/components/_dgo-hero.js]: hadn't done code clean-up yet

File: src/pages/deriv-go/components/_new-app.js (hunk 4 above)
Comment (fiona-deriv) [reply src/pages/deriv-go/components/_new-app.js:66]: okay, I'll add the images

**Pull request closed without merging**